src/main/java/eu/europeana/metis/sandbox/service/workflow/EnrichmentServiceImpl.java

```diff
@@ -24,10 +23,10 @@ public EnrichmentServiceImpl(EnrichmentWorker enrichmentWorker) {
   public RecordInfo enrich(Record record) {
     requireNonNull(record, "Record must not be null");
 
-    String result;
+    byte[] result;
     try {
-      result = enrichmentWorker.process(record.getContentString());
-    } catch (DereferenceOrEnrichException | JiBXException | UnsupportedEncodingException e) {
+      result = enrichmentWorker.process(record.getContentInputStream());
+    } catch (DereferenceOrEnrichException | JiBXException e) {
```

Comment:
JiBXException is never thrown in the try block.

Reply:
That is in fact correct, I will remove it